packages/app-mobile/root.tsx

```diff
@@ -413,6 +414,20 @@ function decryptionWorker_resourceMetadataButNotBlobDecrypted() {
 	ResourceFetcher.instance().scheduleAutoAddResources();
 }
 
+const initializeTempDir = async () => {
+	const tempDir = `${getProfilesRootDir()}/tmp`;
+
+	// Re-create the temporary directory.
+	try {
+		await shim.fsDriver().remove(tempDir);
+	} catch (_error) {
```

Comment:
This try/catch logic is based on the equivalent code used by the desktop (and cli?) apps.